src/ConnectedMode/QualityProfiles/QualityProfileExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Integration;
+
+namespace SonarLint.VisualStudio.ConnectedMode.QualityProfiles
+{
+    internal static class QualityProfileExtensions
+    {
+        public static void LogQPVerbose(this ILogger logger, Language language, string text) => logger.LogVerbose($"[QualityProfile][{language.Name}] {text}");
+
+        public static void LogQPVerbose(this ILogger logger, string text) => logger.LogVerbose("[QualityProfile] " + text);
```

Comment:
Miscellaneous: helper methods so we get consistent prefixes across different QP classes.
Same as for the [MigrationExtensions](https://github.com/SonarSource/sonarlint-visualstudio/blob/154183e913b6a4f2740b601e171835035f20b641/src/ConnectedMode/Migration/MigrationExtensions.cs#L27).